new-client/src/plugins/collector/collector.js

```diff
@@ -28,6 +28,8 @@ class Collector extends React.Component {
         serviceConfig: this.serviceConfig
       }
     });
+
+    this.collectorModel.app = props.app;
```

Comment:
What's the point of this? You already send the app as prop when model is instantiated (line 23). 
```suggestion
// Remove line
```